modules/openapi-generator/src/main/resources/php/libraries/psr-18/api.mustache

```diff
@@ -93,6 +93,14 @@ use function sprintf;
      * @var StreamFactoryInterface
      */
     protected $streamFactory;
+    
+    /**
+     * @var string[]
+     */
+    public const JSON_FORMATS = [
+        'application/merge-patch+json', 
+        'application/json',
+    ]; 
```

Comment:
what about using a regular expression to detect json related MIME type?

https://github.com/OpenAPITools/openapi-generator/pull/19317/files#diff-c1d7ea3a1de60effaa5908cee1be9a7e2fd420bdb75a2e2a28a6c3290380192dR415

Reply:
Sounds good, I updated the code.